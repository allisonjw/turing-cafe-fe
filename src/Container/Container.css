.resy_AllCards {
    display: flex;
    width: 100%;
    flex-wrap: wrap;
    flex-direction: row;
}