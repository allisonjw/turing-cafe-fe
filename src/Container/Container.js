import React from 'react';
import './Container.css';
import Card from '../Card/Card';

const Container = ({reservations}) => {
    let allReservations = reservations.map(res => {
        return (<Card
            key={res.id}
            id={res.id}
            name={res.name}
            date={res.date}
            time={res.time}
            number={res.number} 
        />)
    })
    return (
      <div className="resy_AllCards">
        {allReservations}
      </div>
    )
}

export default Container;