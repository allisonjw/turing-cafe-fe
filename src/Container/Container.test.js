import React from 'react';
import {shallow} from 'enzyme';
import Container from './Container';

describe('Container', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<Container />)
  });

  it.skip('should match all the data being passed', () => {
    expect(wrapper).toMatchSnapshot()
  })

});