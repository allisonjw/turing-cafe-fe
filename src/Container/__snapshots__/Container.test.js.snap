// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container should match all the data being passed 1`] = `undefined`;
